scripts/pylint_extensions_test.py

```diff
@@ -649,8 +654,8 @@ def test_finds_backslash_continuation(self):
         temp_file = tempfile.NamedTemporaryFile()
         filename = temp_file.name
 
-        with open(filename, 'w') as tmp:
-            tmp.write(
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(python_utils.STR(
```

Comment:
Choose between u-strings or wrapping in STR, not both.

Reply:
Done